chainreg/chainparams.go

```diff
@@ -50,6 +50,13 @@ var BitcoinSimNetParams = BitcoinNetParams{
 	CoinType: keychain.CoinTypeTestnet,
 }
 
+// BitcoinSigNetParams contains parameters specific to the signet test network.
+var BitcoinSigNetParams = BitcoinNetParams{
```

Comment:
How are things going for the addition of a new bolt11 invoice prefix?

Reply:
Currently we just use this: `hrp := "ln" + invoice.Net.Bech32HRPSegwit`.
And because the Bech32 HRP part of signet is identical to testnet, you wouldn't be able to tell them apart. Not sure if we should break with the convention. We'd risk losing compatibility with other implementations I think.

Reply:
Oh, I totally expected an extension to the prefixes as defined in https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md for signet. 

Isn't it good to lose compatibility for signet invoices when the sender isn't connected to signet?

Reply:
Hmm, I didn't expect to have a dependency on a spec PR as well... But maybe this could be my first PR there.
What I meant is: If we use a custom prefix in `lnd` only (without adding it to the spec), we'd lose compatibility with the other implementations (for signet).
But yes, I think it would be good to have a different prefix from testnet.

Reply:
The spec meeting is tonight. This sounds like an opportunity to get the fastest merged PR on your name :smile:

Reply:
Okay, let's do this :) https://github.com/lightningnetwork/lightning-rfc/pull/844